docs/download/downloading_data.md

```diff
@@ -0,0 +1,261 @@
+# Downloading Data
+
+## Downloading data from Collaboratory repository
+
+Academic research cloud infrastructure built to house ICGC data.
+
+Repository information can be found [here](https://docs.icgc.org/download/repositories/#collaboratory)
+
+#### Prerequisites
+
+1.  [Apply for DACO access](https://github.com/icgc-dcc/dcc-docs/blob/download-doc-update/docs/download/data-access.md#apply-for-access-to-controlled-data)
+2.  Download and install score-client software [Tarball](/software/download/#score-client), [Docker](https://hub.docker.com/r/overture/score/) (Download/configuration instructions [here](https://docs.icgc.org/download/guide/#installation-of-the-score-client))
+
+<span style="color:blue">NOTE: We are currently actively copying controlled data from [EGA](https://ega-archive.org) to Collaboratory so it can be available to download from either repository. This is an on-going process, so we do not currently have all the EGA datasets in Collaboratory yet.</span>
+
+#### Download Data Instructions
+
+1.  Once your DACO application has been approved, use your OpenID (ie. the Gmail email address you specified in your DACO application) to log into [https://dcc.icgc.org/](https://dcc.icgc.org)
+
+    ![Portal-Login](images/Portal_login.png)
+
+2.  After successful authentication, you will know that you have Cloud Access to the controlled tier if the `Login` link is replaced with a green cloud icon
+
+    ![DACO-Cloud-Access](images/daco-cloud-access.png)
+
+3.  Click on `Token Manager`
+    ![Token-Manager-Link](images/token-manager-link.png)
+
+4.  Select `collab.download` and click `Generate` button to create an access token for downloading data from Collaboratory
+
+    ![Token-Manager](images/token-manager-collab.png)
+
+
+5\. Copy and paste this token into your config file for score-client. Click [here](https://docs.icgc.org/download/guide/#access-configuration) for instructions on how to configure other elements of score-client. The configuration of the Score Client is stored in the `conf/application.properties` file of the distribution. Edit `application.properties` file to add the generated acesss token. 
```

Comment:
Relative URL.

Reply:
why `5\.`, not `5.`?

Reply:
Corrected with relative URL. I used `5\.` instead of `5.` because Markdown resets the numbering after an image or block of code (it resets to `1.` again). So to get around it, I used `5\.`